### PR TITLE
user can download cx_responses by quarter

### PR DESCRIPTION
**Pull request by ryanwoldatwork**
* [x] add specs

### CODE REVIEW

File: app/views/admin/cx_collections/index.html.erb [hunk 1]
```diff
@@ -60,10 +60,10 @@
...
         for
-        <%- if @quarter != 0 %>
+        <%- if @quarter != 0 && @quarter != 'All' %>
         Q<%= params["quarter"] %>
         <% end %>
-        <%- if @year != 0 %>
+        <%- if @year != 0 && @year != 'All' %>
         <%= params["year"] %>
         <% end %>
       </td>
```
Comment (sanason) [inline app/views/admin/cx_collections/index.html.erb:69]: When Year = All or Quarter = All, the second row of the table reports all zeros. Should that be the case?

Screenshot 2024-09-24 at 9 34 12 AM

Screenshot 2024-09-24 at 9 37 22 AM

File: app/views/admin/cx_collections/index.html.erb [hunk 2]
```diff
@@ -60,10 +60,10 @@
     <tr>
       <td class="text-base text-sm">
         for
-        <%- if @quarter != 0 %>
```
Comment (sanason) [inline app/views/admin/cx_collections/index.html.erb:63]: Should this second row not appear if both Year and Quarter are "All"? The bare "for" looks odd.

Screenshot 2024-09-24 at 10 22 15 AM

File: app/controllers/admin/forms_controller.rb [hunk 3]
```diff
@@ -269,6 +269,7 @@ def create
       @form.success_text_heading = t('success')
       @form.success_text = t('form.submit_thankyou')
       @form.delivery_method = 'touchpoints-hosted-only'
+      @form.kind = 'custom'
```
Comment (ryanwoldatwork) [inline app/controllers/admin/forms_controller.rb:272]: set new form.kind to `custom`

File: app/views/admin/cx_collections/index.html.erb (hunk 2 above)
Comment (ryanwoldatwork) [reply app/views/admin/cx_collections/index.html.erb:63]: Screenshot 2024-10-16 at 10 33 36 AM

**Review by sanason — APPROVED** [review]
LGTM. Left one comment.

File: app/models/form.rb [hunk 4]
```diff
@@ -51,6 +52,25 @@ def self.filtered_forms(user, aasm_state)
...
+      "a11_v2", # launched fall 2023
+      "a11_yes_no",
+      "open_ended",
+      "other", # TODO: deprecate in favor of custom,
+      "recruiter",
+      "yes_no",
+      "custom"
+    ]
+  end
+
```
[the remark this answers is not in the document]
Comment (ryanwoldatwork) [reply app/models/form.rb:67]: can/will validate forms in coordination with deploy

**Pull request merged**